### ISSUE TITLE
Request: Custom File Explorer sorting compatibility

### ISSUE THREAD

**Issue by dantveita** [pjeby/quick-explorer#100]
Is it possible to make it compatible with the Custom File explorer sorting plugin? (https://github.com/SebastianMC/obsidian-custom-sort) 

That is; I mainly use this plugin to navigate file tree with keyboard shortcuts. And I want to be able to both pin notes to the top of my root file tree, as well as sort my files by note creation date. 

When invoking the Quick Explorer, im able to navigate my custom sort order properly. But when using the hotkeys for next/previous note, the custom order is not respected.

Any way to fix that?

**Comment by pjeby** [comment]
Not really, no.  As far as I can tell, that plugin does not offer a way to share its sort information or status with other plugins, so I would have to duplicate its functionality of reading config files, bookmarks, etc.
